<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playPauseButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="processSlider">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>清空列表</string>
   </property>
  </widget>
  <widget class="QListWidget" name="songsList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addSongsButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>添加歌曲</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>删除歌曲</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalTimeLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="presentTimeLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="volumeButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
